Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:argon
```

Comment:
Yeah it makes to try and build everything rougly similar unless there's a good reason not to.